 Customize live image for oe:

 * disable ssl verification
 * customize repo path

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/core/constants.py                       | 2 +-
 pyanaconda/modules/common/structures/live_image.py | 2 +-
 pyanaconda/modules/payloads/source/cdrom/cdrom.py  | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/pyanaconda/core/constants.py b/pyanaconda/core/constants.py
index b2854196b..ca657e5d4 100644
--- a/pyanaconda/core/constants.py
+++ b/pyanaconda/core/constants.py
@@ -52,7 +52,7 @@ DRACUT_ISODIR = "/run/install/source"
 ISO_DIR = MOUNT_DIR + "/isodir"
 IMAGE_DIR = MOUNT_DIR + "/image"
 INSTALL_TREE = "/media/source"
-SOURCES_DIR = MOUNT_DIR + "/sources"
+SOURCES_DIR = "/run/install/sources"
 BASE_REPO_NAME = "anaconda"
 
 # Get list of repo names witch should be used as base repo
diff --git a/pyanaconda/modules/common/structures/live_image.py b/pyanaconda/modules/common/structures/live_image.py
index b47e6252c..6b3d37862 100644
--- a/pyanaconda/modules/common/structures/live_image.py
+++ b/pyanaconda/modules/common/structures/live_image.py
@@ -30,7 +30,7 @@ class LiveImageConfigurationData(DBusData):
         self._url = ""
         self._proxy = ""
         self._checksum = ""
-        self._ssl_verification_enabled = True
+        self._ssl_verification_enabled = False
 
     @property
     def url(self) -> Str:
diff --git a/pyanaconda/modules/payloads/source/cdrom/cdrom.py b/pyanaconda/modules/payloads/source/cdrom/cdrom.py
index f1def7979..dd6c3fa13 100644
--- a/pyanaconda/modules/payloads/source/cdrom/cdrom.py
+++ b/pyanaconda/modules/payloads/source/cdrom/cdrom.py
@@ -93,7 +93,7 @@ class CdromSourceModule(PayloadSourceBase, MountingSourceMixin, RPMSourceMixin):
 
     def generate_repo_configuration(self):
         """Generate RepoConfigurationData structure."""
-        return RepoConfigurationData.from_directory(self.mount_point)
+        return RepoConfigurationData.from_directory(self.mount_point + '/Packages')
 
     def set_up_with_tasks(self):
         """Set up the installation source.
